### ISSUE TITLE
consider leap years for year offset

### ISSUE THREAD

**Issue by thomas-maschler** [wri/mapnik-forest-change-tiles#2]
@mappingvermont I am starting to review the GLAD repos.
https://github.com/wri/mapnik-forest-change-tiles/blob/0d2d3dd5368ddb152b2ed27f5ce53e4b253a98be/raster_processing/utilities/gdal_prep_date_conf.py#L28
Why do we use 365 for year_offset? What if there is a leap year? 
We could do `365 + int(not(year%4))` instead.

I assume this would this would affect the current frontend implementation and we would need to process 2016 and 2017 again, right?

**Comment by mappingvermont** [comment]
Yeah - good point. I'm not sure why I didn't include leap year in that calculation. Might want to check with the UMD folks to see if they factor it in- I'm assuming that they do. And yeah- 2016 and 2017 would need to be reprocessed in that case.

Good spot! Let me know if you have other questions - hope everything else is working smoothly.
